app/src/app/components/header/header.component.html

```diff
@@ -41,6 +42,7 @@ <h1><a routerLink="/">ATT&CK Workbench&nbsp;<span class="app-version lowercase">
                 [routerLink]="stixRoute.path">{{stixRoute.data.breadcrumb}}</button>
             <button mat-menu-item routerLink="/docs">documentation</button>
             <button mat-menu-item (click)="isLoggedIn ? logout() : login()">{{isLoggedIn ? 'logout': 'login'}}</button>
+            <button mat-menu-item *ngIf="!isLoggedIn" (click)="register()">register</button>
```

Comment:
Same comment as above. The register button should be hidden with `anonymous` authentication.